lib/import/colorectal/providers/st_george/st_george_handler_colorectal.rb

```diff
@@ -0,0 +1,317 @@
+require 'possibly'
+require 'pry'
```

Comment:
I don't think this is needed?

Reply:
updated in [dfsdf](https://github.com/NHSDigital/data_management_system/commit/b64fd4c1b851661d098425a1ab1df681db92063f)

Reply:
removed in [9008c0c](https://github.com/NHSDigital/data_management_system/commit/9008c0cbeb9b18d771b626be85f1a2bf7f77a437)